Library/Homebrew/formula_support.rb

```diff
@@ -28,13 +28,14 @@ def valid?
 
   def to_s
     return @explanation unless @explanation.empty?
+    os_name = OS::Mac.os_name(ENV["HOMEBREW_OSX_VERSION"])
```

Comment:
@MikeMcQuaid Yep, I mean requiring `os/mac` on `test_formula_support`, just like what we do in `test_os_mac_language`.